src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final String DEFAULT_LOGIN = "valid_login";
+    private static final String DEFAULT_PASSWORD = "valid_password";
+    private static final int DEFAULT_AGE = 25;
+    private User user;
+
+    @BeforeAll
+    static void testUserCreation() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setLogin(DEFAULT_LOGIN);
+    }
```

Comment:
You need to do this logic in each tests. Not in setUp

Reply:
I'm not agree with you, why should I make more code when I can use BeforeEach? I think this is more optimized solution than creating new user in each test